--- atop-1.23/atop.init.LSB	2008-03-06 11:15:47.000000000 +0100
+++ atop-1.23/atop.init	2009-11-10 18:19:05.000000000 +0100
@@ -5,6 +5,11 @@
 # chkconfig: 2345 85 15
 # description:	Atop is a system and process activity monitor
 #
+### BEGIN INIT INFO
+# Provides: atop
+# Short-Description: Launch atop monitor
+# Description: Launch atop monitor
+### END INIT INFO
 
 # Check existance of binaries 
 [ -f /usr/bin/atop ] || exit 0
